Automattic/wp-calypso
# Live Preview: remove the welcome modal

Related to:

- https://github.com/Automattic/wp-calypso/issues/80264


## Proposed Changes

This removes the old Live Preview Modal as it is replaced by https://github.com/Automattic/jetpack/pull/34935.

This should ONLY be deployed shortly before we release the above PR.

Note: we tried hiding this modal when showing the new modal but it does not always work, so we removed the code via https://github.com/Automattic/jetpack/pull/34935/commits/85c2cc2725567b28eed43f6245120674521b86f0.

## Testing Instructions

1. Sandbox your site.
2. Patch this: `install-plugin.sh etk live-preview-modal-remove`
3. Live-preview a theme; verify that no modal is shown.

## Pre-merge Checklist
